adafruit_thermal_printer/thermal_printer.py

```diff
@@ -309,14 +319,14 @@ def print_barcode(self, text, barcode_type):
         self.feed(1)  # Recent firmware can't print barcode w/o feed first???
         self.send_command("\x1DH\x02")  # Print label below barcode
         self.send_command("\x1Dw\x03")  # Barcode width 3 (0.375/1.0mm thin/thick)
-        self.send_command("\x1Dk{0}".format(chr(barcode_type)))  # Barcode type
+        self.send_command(f"\x1Dk{chr(barcode_type)}")  # Barcode type
         # Write length and then string (note this only works with 2.64+).
         self.send_command(chr(len(text)))
         self.send_command(text)
         self._set_timeout((self._barcode_height + 40) * self._dot_print_s)
         self._column = 0
 
-    def _print_bitmap(self, width, height, data):
+    def _print_bitmap(self, width: int, height: int, data: bytes) -> None:
```

Comment:
`data` seems like it could be any readable buffer, so let's typing it as `circuitpython_typing.ReadableBuffer`.